### DISCUSSION TITLE
Assigning morph id to bots

### DISCUSSION THREAD

**Discussion by westernoutlaw** [trickerer/Trinity-Bots#716] [Ideas]
Just a question/idea, I honestly don't even know whether it's possible or not.
I enjoy morphing bots to various displayids, which I do manually through a macro with .morph, for example I morph my tank into abomination, healer into banshees, etc. So I was thinking, would it be possible to add an option to assign a permanent displayid (and possibly scale too) when creating a bot? As far as I can tell, most of them don't have any issues with class abilities, and I think that would add a lot of depth to the mod overall

**Comment by trickerer** [comment]
It's possible, but only if you disable equipment display in config (doesn't affect weapons). Set desired bot's display id in `creature_template` table and that should be it. Oh and delete client's cache.
Unfortunatelly doing this separately is no longer possible without recompile since I had to move mirror image flag from table to code.
You may also try to just set bot's display id in `creature_template` table but result may look ugly if the model for some reason supports player model visual slots.

There is also a `creature_template_outfits` table which is used to set in stone visuals for certain bots or playable races (per creature id, not class). Currently it only has entries for Blademaster (and Mirror Image). Values there take priority over any other equipment expect weapon / shield, the thing I'm not sure about is if equip slot values are item ids or item display ids. Spawn Blademaster to see their visuals (don't hire, they are buggy)
